app/src/main/java/com/github/wnder/GuessLocationActivity.java

```diff
@@ -221,6 +250,23 @@ public void run() {
         };
         timer = new Timer(true);
 
+        //Setup image preview
+        //littleImage, encapsulated in littleCard, is the image shown when zoomed out
+        //bigImage, encapsulated in bigCard, is the image shown when zoomed in
+        littleCard = findViewById(R.id.imageToGuessCard);
+        bigCard = findViewById(R.id.imageToGuessCardZoomedIn);
+        bigCard.setVisibility(View.INVISIBLE);
```

Comment:
You can just write VISIBLE/INVISIBLE (without View)

Reply:
I just tried to be consistent with the rest of the class that was already written, but now I changed everything